UIViewController-DisplayChild/UIViewController+DisplayChild.swift

```diff
@@ -102,7 +102,17 @@ public extension UIViewController {
 
     func childViewController<T>(in optionalContainerView: UIView? = nil) -> T? {
         let containerView = optionalContainerView ?? view!
-        return childViewController(at: containerView) as? T
+        let result: T?
+        //see also `presentedController(with:)
+        if let controller = childViewController(at: containerView) as? T {
+            result = controller
+        } else if let navigationController = childViewController(at: containerView) as? UINavigationController,
```

Comment:
The naming is ambiguous for these functions:

```swift
func childViewController<T>(in ...)
private func childViewController(at ...)
```
Just consider the call site:
```swift
childViewController(at: containerView)
vs.
childViewController(in: containerView)
```
I know that this was not introduced in this MR. Anyway, I just want to point it out.

Reply:
Maybe it is not that confusing, because you have to provide type on calling site. And compiler tells you this. I am not saying this is ideal, though. Maybe rename the private one to `anyChildViewController(in: containerView)`, but I do not like this either.
```Swift
let controller: TestViewController = childViewController(in: containerView)
let controller = childViewController(at: containerView)
```